Views/BuildingBlocks/GridSection.swift

```diff
@@ -29,8 +29,8 @@ struct GridSection<Content: View>: View {
 
 struct GridSection_Previews: PreviewProvider {
     static var previews: some View {
```

Comment:
This is a preview section, not production part of the application, no translations needed.